src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
 content_type: explanation
 ---
 
-In addition to the {{site.base_gateway}} open source, free, and plus features, users with an Enterprise plan also have access to Enterprise-only features. These features enhance the security, monitoring, and management capabilities of {{site.base_gateway}}.
+Kong Enterprise is the scalable, secure, and flexible API management solution that extends Kong Gateway, the fastest, most adopted API gateway, with enterprise plugins, developer portal, analytics, advanced security features, GUI's, and 24/7 support. It is the only solution that helps you accelerate your cloud journey by managing, securing, and monitoring connections between applications across hybrid and multi-cloud architectures, to help you scale faster and boost developer productivity. 
 
-The following sections describe key Kong Enterprise features. 
+## Enterprise Plugins
+
+{{site.base_gateway}} offers access to 400+ out-of-box enterprise and community plugins. {{site.base_gateway}} Enterprise offers exclusive versions of OSS plugins like the [Rate-Limiting Advanced plugin](hub/kong-inc/rate-limiting-advanced/) with added functionality such as the use of consumer groups, and database specific strategy. {{site.base_gateway}} Enterprise natively support gRPC and REST, WebSockets and integrate with Apollo GraphQL server and Apache Kafka services. These plugins can be leveraged to provide advanced connectivity features and solutions to {{site.base_gateway}} such as:
```

Comment:
The Rate Limiting plugin link is a 404. Not sure why because the URL looks correct... When I click on it, it takes me to "/gateway/latest/kong-enterprise/hub/kong-inc/rate-limiting-advanced/"

Reply:
It's missing the leading / at the start of the link.